packages/webapp/constants/types.ts

```diff
@@ -11,10 +11,10 @@ export const enum CHAIN_IDS {
   BASE_SEPOLIA = 84532
 }
 
-export const ALCHEMY_RPC_URLS: Map<number, string> = new Map([
-  [CHAIN_IDS.ETHEREUM, "https://eth-mainnet.g.alchemy.com/v2/"],
-  [CHAIN_IDS.BASE, "https://base-mainnet.g.alchemy.com/v2/"]
-])
+export const ALCHEMY_RPC_URLS= {
```

Comment:
tbh this and CHAIN_IDS doesn't seem like it belongs in a "types" file

Reply:
I thought, and think the same. It's how it's set up in `nouns-builder` so it makes all the dozens and dozens of import changes easier while getting all the other connections working. I've been using it the same way with auctions. I think once auctions/proposals are in working order we can revisit with a find replace, but is worth it to keep as is for now. Can you create an issue for revisiting?